### PR TITLE
Update rocket hash

### PR DESCRIPTION
**Pull request by hmtanbir**
Change type meta tag in templates to Hash 1.9 syntax

### CODE REVIEW

**Comment by hmtanbir** [comment]
@pirj I have checked and make the PR again.

**Comment by pirj** [comment]
By what I meant in https://github.com/rspec/rspec-rails/pull/2198#issuecomment-548331420 is to go through https://github.com/hmtanbir/rspec-rails/tree/ed181315e9a4dfc257bfd108f491d9092c56f567/lib/generators/rspec/scaffold/templates and https://github.com/hmtanbir/rspec-rails/blob/ed181315e9a4dfc257bfd108f491d9092c56f567/lib/generators/rspec/scaffold/scaffold_generator.rb and replace the old hash syntax with the new one.

As far as I understand, you're scratching your own itch with this change, i.e. you're using a linter in your project and it complains about the old hash syntax. Is this the reasoning behind this change?

File: lib/generators/rspec/scaffold/templates/routing_spec.rb [hunk 1]
```diff
@@ -5,41 +5,41 @@
...
     end
 
     it "routes to #update via PUT" do
-      expect(:put => "/<%= ns_table_name %>/1").to route_to("<%= ns_table_name %>#update", :id => "1")
+      expect(put: "/<%= ns_table_name %>/1").to route_to("<%= ns_table_name %>#update", :id => "1")
     end
 
     it "routes to #update via PATCH" do
-      expect(:patch => "/<%= ns_table_name %>/1").to route_to("<%= ns_table_name %>#update", :id => "1")
+      expect(patch: "/<%= ns_table_name %>/1").to route_to("<%= ns_table_name %>#update", :id => "1")
```
Comment (pirj) [inline lib/generators/rspec/scaffold/templates/routing_spec.rb]: This is mixing the old and the new hash style syntaxes.

Comment (hmtanbir) [reply lib/generators/rspec/scaffold/templates/routing_spec.rb]: @pirj I have updated the files according to your feedback and you are correct, robocop of my project complains about the old hash tag and I had to change every time.

**Comment by pirj** [comment]
To avoid build failures please feel free to update `Style/HashSyntax` to prefer the new hash syntax.

**Comment by pirj** [comment]
@hmtanbir Are you up to wrap this change up?
What's left is https://github.com/rspec/rspec-rails/pull/2199#issuecomment-548352394 and `.rubocop.yml`.

**Comment by pirj** [comment]
@hmtanbir Please feel free to reopen if you decide to update all the templates.

**Pull request closed without merging**